### ISSUE TITLE
Add no-expire option?

### ISSUE THREAD

**Issue by rochacbruno** [heltonalves99/quickjson#7]
Why expiration?

Cant handle the storage compressed and keep it forever?

**Comment by fidelisrafael** [comment]
Good point, Redis can store persisted data to disk too, and anyway we have better NoSQL alternatives.

**Comment by rochacbruno** [comment]
The deploy can be done to OpenShift free account and there is MongoDB support there.
